portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase

```diff
@@ -2488,4 +2488,107 @@ definition {
 		FormFields.viewTextLabel(fieldLabel = "Text");
 	}
 
+	@description = "Case 1: Validating a String"
+	@priority = 5
+	test ValidateConfigurationStringAtControlPanel {
+		ApplicationsMenuHelper.openApplicationsMenu();
+
+		ApplicationsMenu.gotoPanel(panel = "Control Panel");
+
+		AssertElementPresent(
+			locator1 = "//h2[@id='nav_control_panel.configuration']");
+	}
+
+	@description = "Case 2: Writing tests in Forms"
```

Comment:
See your description! If I was someone who don't know nothing about your code,  your description wouldn't help me understand what you doing. See other testcase and type a new @description.